README.md

```diff
@@ -25,72 +25,72 @@ The configuration is being read on load from many sources, in the following orde
 
 #### url
 
-Type: string  
-Default: `'http://localhost:5984'`  
+Type: string
+Default: `'http://localhost:5984'`
```

Comment:
The spaces are intentional. It's to force a new line in the rendering. You can also use `<br>` if you think it's clearer.